percona/controllers/pgc/pgc.go

```diff
@@ -152,12 +170,12 @@ func (c *Controller) onAdd(obj interface{}) {
 
 	err = c.handleScheduleBackup(newCluster, nil)
 	if err != nil {
-		log.Errorf("handle schedule: %s", err)
+		log.Errorf("handle schedule backups: %s", err)
 	}
 	ctx := context.TODO()
 	_, err = c.Client.CrunchydataV1().PerconaPGClusters(newCluster.Namespace).Update(ctx, newCluster, metav1.UpdateOptions{})
 	if err != nil {
-		log.Errorf("handle schedule: %s", err)
+		log.Errorf("update perconapgcluster: %s", err)
```

Comment:
why we're continuing execution if error?

Reply:
https://github.com/percona/percona-postgresql-operator/pull/181/commits/ed0fd46c2efa1c89d88cd96a34961a807fa40913